http_server/_test_headers.pony

```diff
@@ -1,6 +1,7 @@
 use "debug"
 use "pony_check"
```

Comment:
This is unrelated at this point, no new uses were added so I think this should be rolled back